vite.config.ts

```diff
@@ -25,9 +25,6 @@ export default defineConfig({
   },
   // Temporary "fix" https://github.com/storybookjs/storybook/issues/28542
   optimizeDeps: {
-    exclude: [
-      './apps/storybook/node_modules/.cache/storybook',
-      './apps/storybook/docs-components',
```

Comment:
Why was docs-components excluded?

Reply:
This was here to fix an issue where storybook would crash when importing these components.
Seems like this has been fixed, since the issue is no long occuring.

Reply:
alrighty, lets see how it goes without!